dvc/commands/data.py

```diff
@@ -0,0 +1,178 @@
+import argparse
+import logging
+from typing import TYPE_CHECKING
+
+from funcy import compact, log_durations
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link, fix_subparsers
+from dvc.ui import ui
+
+if TYPE_CHECKING:
+    from dvc.repo.data import Status as DataStatus
+
+
+logger = logging.getLogger(__name__)
+
+
+class CmdDataStatus(CmdBase):
+    COLORS = {
+        "not_in_cache": "red",
+        "committed": "green",
+        "uncommitted": "yellow",
+        "untracked": "cyan",
+    }
+    LABELS = {
+        "not_in_cache": "Not in cache",
+        "committed": "DVC committed changes",
+        "uncommitted": "DVC uncommitted changes",
+        "untracked": "Untracked files",
+        "unchanged": "DVC unchanged files",
+    }
+    HINTS = {
+        "not_in_cache": 'use "dvc pull <file>..." '
+        "to update your local storage",
+        "committed": "git commit the corresponding dvc files "
+        "to update the repo",
+        "uncommitted": 'use "dvc commit <file>..." to track changes',
+        "untracked": 'use "git add <file> ..." or '
+        'dvc add <file>..." to commit to git or to dvc',
+        "git_dirty": "there are {}changes not tracked by dvc, "
+        'use "git status" to see',
```

Comment:
```suggestion
        "git_dirty": "There are {}changes not tracked by DVC. "
        'Use `git status` to see them.',
```

Reply:
It's a hint, no need to upper case it.

Reply:
I don't understand why hints don't need capitalization or punctuation. Are they part of a more complete sentence?